src/quiz/quiz.stories.tsx

```diff
@@ -235,4 +333,115 @@ const App = () => {
 	},
 };
 
+export const WithValidationAndAnswerFeedback: Story = {
+	render: QuizWithValidationAndAnswerFeedback,
+	args: {},
+	parameters: {
+		docs: {
+			source: {
+				code: `
+import { Quiz, useQuiz, Button, Spacer } from '@freecodecamp/ui';
+
+const initialQuestions = [
+  {
+		question: "Lorem ipsum dolor sit amet",
+		answers: [
+			{
+				label: "Option 1",
+				value: 1,
+				feedback: (
+					<PrismFormatted
+						text={\`<p>Quaerat in autem sapiente illum. Vel mollitia omnis qui dolorem <code>um</code> esse eos maiores possimus. Est laborum quam aliquam qui sunt. Ut ea et qui provident voluptatibus. Eius quam odit sint cumque sint. Corporis quia et dicta.</p>\`}
+						getCodeBlockAriaLabel={(codeName) => \`\${codeName} code example\`}
+					/>
+				),
+			},
+			{
+				label: "Option 2",
+				value: 2,
+				feedback:
+					"Recusandae necessitatibus consequatur voluptatem sapiente.",
+			},
+			{ label: "Option 3", value: 3, feedback: "Voluptas et et animi quo." },
+		],
+		correctAnswer: 1,
+	},
+	{
+		question: "Consectetur adipiscing elit",
+		answers: [
+			{
+				label: "Option 1",
+				value: 1,
+				feedback: (
+					<PrismFormatted
+						text={\`<p>Quaerat in autem sapiente illum. Vel mollitia omnis qui dolorem <code>um</code> esse eos maiores possimus. Est laborum quam aliquam qui sunt. Ut ea et qui provident voluptatibus. Eius quam odit sint cumque sint. Corporis quia et dicta.</p>\`}
+						getCodeBlockAriaLabel={(codeName) => \`\${codeName} code example\`}
+					/>
+				),
+			},
+			{
+				label: "Option 2",
+				value: 2,
+				feedback:
+					"Recusandae necessitatibus consequatur voluptatem sapiente.",
+			},
+			{ label: "Option 3", value: 3, feedback: "Voluptas et et animi quo." },
+		],
+		correctAnswer: 2,
+	},
+	{
+		question: "Fugit itaque delectus voluptatem alias aliquid",
+		answers: [
+			{
+				label: "Option 1",
+				value: 1,
+				feedback: (
+					<PrismFormatted
+						text={\`<p>Quaerat in autem sapiente illum. Vel mollitia omnis qui dolorem <code>um</code> esse eos maiores possimus. Est laborum quam aliquam qui sunt. Ut ea et qui provident voluptatibus. Eius quam odit sint cumque sint. Corporis quia et dicta.</p>\`}
+						getCodeBlockAriaLabel={(codeName) => \`\${codeName} code example\`}
+					/>
+				),
+			},
+			{
+				label: "Option 2",
+				value: 2,
+				feedback:
+					"Recusandae necessitatibus consequatur voluptatem sapiente.",
+			},
+			{ label: "Option 3", value: 3, feedback: "Voluptas et et animi quo." },
+		],
+		correctAnswer: 3,
+	},
+];
+
+const App = () => {
+  const { questions, validateAnswers } = useQuiz({
+	  initialQuestions,
+    validationMessages: {
+      correct: "Correct.",
+      incorrect: "Incorrect.",
+	  },
+	});
+
+  const [disabled, setDisabled] = useState(false);
+
+  const handleSubmit = () => {
+    validateAnswers();
+    setDisabled(true);
+  };
+
+  return (
+    <div>
+      {correctAnswerCount && <p>Correct answers: {correctAnswerCount}</p>}
```

Comment:
```suggestion
      {!!correctAnswerCount && <p>Correct answers: {correctAnswerCount}</p>}
```
Same issue as above.